spec/models/web_results_post_processor_spec.rb

```diff
@@ -81,7 +81,7 @@
         @post_processed_results = post_processor.post_processed_results
       end
 
-      it "should replace Bing's result title with the NewsItem title" do
+      it "replaces Bing's result title with the NewsItem title" do
```

Comment:
while you'r in here make this single quote please.

Reply:
This needs to be double-quoted due to the apostrophe in `Bing's`.

Reply:
👍